hack/jenkins/e2e-runner.sh

```diff
@@ -82,6 +82,12 @@ function fetch_server_version_tars() {
     unset CLUSTER_API_VERSION
 }
 
+function fetch_gci_version_tars() {
+    echo "Using GCI builtin version: ${GCI_K8S_VERSION}"
```

Comment:
It's a bit nasty that the definition of `GCI_K8S_VERSION` is so far from its usage.